firebase_admin/_auth_utils.py

```diff
@@ -385,6 +385,15 @@ def __init__(self, message, cause=None, http_response=None):
         exceptions.NotFoundError.__init__(self, message, cause, http_response)
 
 
+class UserDisabledError(exceptions.FailedPreconditionError):
```

Comment:
Since revoked token error extends `exceptions.InvalidArgumentError`, I have updated `UserDisabledError` to also extend `exceptions.InvalidArgumentError`.

I prefer keeping the error in `_auth_utils.py` since there is also a backend error code `USER_DISABLED` which is currently used for client operation errors but could be re-used in a different context for the admin sdk.